### PR TITLE
Update agenda

### PR DESCRIPTION
**Pull request by maximilianigl**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -79,56 +79,63 @@ organizers_row:
...
+    content: "Frontiers in Embodied AI for Autonomous Driving"
   - time: "16:30"
     image_path: "assets/images/speakers/kashyap_chitta.jpeg"
     person_name: Kashyap Chitta
     person_affiliation: University of Tübingen
-    content: "tbd"
+    content: "Synthesizing Simulation Environments with Generative Models"
+  - time: "17:00"
+    event: "Panel Discussion"
+    content: Dr. Raquel Urtasun, Dr. Dragomir Anguelov, Dr. Jamie Shotton, and Dr. Gustav Markkula"
```
Comment (Thomas0Gilles) [inline index.md]: Not that it matters but I think Raquel and Gustav are professors

**Review by Thomas0Gilles — DISMISSED** [review]

File: index.md (hunk 1 above)
Comment (m-naumann) [reply index.md]:
```suggestion
    content: Raquel Urtasun, Dragomir Anguelov, Jamie Shotton, and Gustav Markkula"
```
I'd leave out Dr./Prof., as  elsewhere

File: index.md [hunk 2]
```diff
@@ -176,8 +183,8 @@ work in archival journals or conferences.
 
 **Important Dates:**
 * Submission deadline: April 5th
-* Notification to authors: ~~April 19th~~ April 24th
-* Camera ready deadline: ~~April 26th~~ May 1st
+* Notification to authors: April 19th
+* Camera ready deadline: April 26th
```
Comment (m-naumann) [inline index.md]: was this intentional? We could change to
```suggestion
* Notification to authors: April 24th
* Camera ready deadline: May 1st
```

File: index.md (hunk 1 above)
Comment (maximilianigl) [reply index.md]: Changed it

File: index.md (hunk 2 above)
Comment (maximilianigl) [reply index.md]: Ah, wasn't intentional, I messed up the branching commit. 
Fixed it now.

**Review by m-naumann — APPROVED** [review]

**Pull request merged**